apps/api/src/applicant/ienapplicant.util.service.ts

```diff
@@ -151,15 +152,20 @@ export class IENApplicantUtilService {
     applicant: IENApplicant,
     dataToUpdate: any,
     job: IENApplicantJob | null,
-  ): Promise<IENApplicantStatusAudit> {
+  ): Promise<IENApplicantStatusAudit | any> {
     // Save
     const status: Partial<IENApplicantStatusAudit> = {
       applicant: applicant,
       job: job,
       ...dataToUpdate,
     };
-    const status_audit = this.ienapplicantStatusAuditRepository.create(status);
-    return this.ienapplicantStatusAuditRepository.save(status_audit);
+
+    await getManager().transaction(async manager => {
+      const status_audit = manager.create<IENApplicantStatusAudit>(IENApplicantStatusAudit, status);
```

Comment:
`create` just creates an entity to be saved, so these two lines are just one operation.